src/assets/feature-flags.json

```diff
@@ -0,0 +1,417 @@
+[
+  {
+    "id": "help",
+    "developmentState": "stable"
+  },
```

Comment:
I initially introduced alphabetical order here, in case you didn't notice, plz keep it sorted to simplify navigation.